### PR TITLE
Merge master to 0.10.1

### PR DESCRIPTION
**Pull request by twoeths**
## Goal
+ I need this in order to work on #707

### CODE REVIEW

File: packages/lodestar-params/src/presets/minimal.ts [hunk 1]
```diff
@@ -1,70 +1,12 @@
...
-export const MAX_VOLUNTARY_EXITS = 2 ** 4; // 16
+// eslint-disable-next-line @typescript-eslint/ban-ts-ignore
+// @ts-ignore
+import minimalYaml from "./minimal.yaml";
+import {convertTypes, schema} from "../utils";
+import {typeMap} from "../types";
+import {load} from "js-yaml";
+
+
+// TODO: fix this
```
Comment (twoeths) [inline packages/lodestar-params/src/presets/minimal.ts]: `GENESIS_FORK_VERSION` is not loaded correctly, will create an issue for it

**Comment by twoeths** [comment]
Unit tests are not passed, investigating ...

**Review by mpetrunic — APPROVED** [review]

File: packages/lodestar-params/src/presets/minimal.ts (hunk 1 above)
Comment (mpetrunic) [reply packages/lodestar-params/src/presets/minimal.ts]: just remove it from [constants](https://github.com/ChainSafe/lodestar/blob/master/packages/lodestar-params/src/constants.ts) as it was not in config before

Comment (twoeths) [reply packages/lodestar-params/src/presets/minimal.ts]: it works, thanks

**Pull request merged**